tests/test_segmentation.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+from unittest.mock import patch
+
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+import napari
+
+import epitools._sample_data
+from epitools.analysis import calculate_segmentation
+
+SPOT_SIGMA = 4.0
+OUTLINE_SIGMA = 3.0
+THRESHOLD = 3.0
+
+
+@pytest.fixture(scope="function")
+def projected_image() -> napari.layers.Image:
+    data, kwargs, layer_type = epitools._sample_data.load_projected_data()[0]
+    return napari.layers.Image(data, **kwargs)
+
+
+@pytest.fixture(scope="function")
```

Comment:
Just spotted this `fixture` (and possibly others) is exactly the same as the one in `test_cell_statistics.py`. You should make a `conftest.py` and put `fixture`s in there. Then it is easy to know where they come from and no need for repetition. See e.g. https://github.com/astro-informatics/sleplet/blob/main/tests/conftest.py if you're not sure what I'm on about.

Reply:
yeah there's a lot of overlap. There were two fixtures that had the same name but returned different things (well, they both returned a viewer, but with different layers added). So I've renamed them and added all fixtures to `conftest.py`